addons/web_editor/static/tests/link_tests.js

```diff
@@ -93,7 +93,7 @@ QUnit.module(
             await nextTick();
             assert.strictEqual(
                 editable.innerHTML,
-                `<p><a href="#" target="_blank">#</a><br></p>`
+                `<p><a href="#" target="_blank"><span data-o-link-zws="start" contenteditable="false">\u200B</span>#</a><span data-o-link-zws="after" contenteditable="false">\u200B</span><br></p>`
```

Comment:
Try calling `editor.clean();` before like some tests below already does to avoid the zws getting in the way of the expected result.

Reply:
I have updated the test cases as suggested.